felix/dataplane/linux/int_dataplane.go

```diff
@@ -796,6 +797,10 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 	bpfconntrack.SetCleanupMapSize(config.BPFMapSizeConntrackCleanupQueue)
 	bpfifstate.SetMapSize(config.BPFMapSizeIfState)
 
+	if config.BPFMapSizePerCPUConntrack > 0 {
+		bpfconntrack.SetMapSize(config.BPFMapSizePerCPUConntrack * bpfmaps.NumPossibleCPUs())
```

Comment:
Please adjust so that `bpfconntrack.SetMapSize` is only called once after calculating which of the two settings should apply.  Easy to miss this second call when reading the code later.

Reply:
Good catch! Will adjust as you suggested ;)